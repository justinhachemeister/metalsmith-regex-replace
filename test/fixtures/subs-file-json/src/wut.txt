Teh game said "A winner is yuo!"