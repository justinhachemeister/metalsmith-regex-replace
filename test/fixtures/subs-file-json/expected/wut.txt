The game said "A winner is you!"